collector/src/main/java/com/bitmovin/analytics/license/DefaultLicenser.java

```diff
@@ -0,0 +1,17 @@
+package com.bitmovin.analytics.license;
+
+public class DefaultLicenser implements OnAuthCompleted, Licenser {
```

Comment:
`Licenser` is more of a `go` name than a Java name. A better name for Java would be a `LicenseProvider` or smth similar

Reply:
Agreed.